seacatauth/events.py

```diff
@@ -0,0 +1,28 @@
+class EventTypes:
+	EVENT_TYPE = "event_type"
```

Comment:
i see what you mean, but this is semantically not an event type so it's weird to have it here. you could possibly call it `FIELD_NAME` and the class `EventType` (singular) so that when you call it it makes more sense (`EventType.FIELD_NAME`).

as a follow-up, you can create an issue in ASAB: we could add `event_type` as an optional argument to the `upsertor.execute()` method, to standardize it and to avoid having this field here altogether.